src/vm/runners/cairo_runner.rs

```diff
@@ -237,6 +238,35 @@ impl CairoRunner {
         self.relocate_memory(&relocation_table);
         self.relocate_trace(&relocation_table);
     }
+
+    pub fn print_output(&self, stdout: &mut dyn io::Write) {
```

Comment:
The problem with using print! is that we can't easily capture the standard output to use in tests, which is very helpful. Using write!, we can send a Vec instead of stdout to test the program output.

Reply:
that makes sense, but then let's change the function name @fmoletta